conf/pipeline/sarek/cfc.config

```diff
@@ -0,0 +1,24 @@
+// Profile config names for nf-core/configs
+
+params {
+  // Specific nf-core/configs params
+  config_profile_contact = 'Friederike Hanssen (@FriederikeHanssen)'
+  config_profile_description = 'nf-core/sarek CFC profile provided by nf-core/configs'
+}
+
+// Specific nf-core/sarek process configuration
+process {
+  withName:'StrelkaSingle|Strelka|StrelkaBP|MantaSingle|Manta' {
+    memory = 128.GB
```

Comment:
I so much memory necessary? Basically anything over 60.GB will go to `qbic` queue and occupy the high-mem nodes

Reply:
Fair enough. For an initial attempt I reduced anything that was labeled with `max_cpus/memory` to whatever max setting I had. I think, we can risk going much lower and just increase, if too many people report failures. I'd be happy to keep taps on that and increase if needed.

Reply:
So for now maybe everything to CFC queue with 59Gb of memory and 19 cpus and then evaluate over a few weeks whether any significant problems or delays occur?

Reply:
that sounds good, thanks!